includes/widgets/icon-list.php

```diff
@@ -296,14 +296,24 @@ protected function register_controls() {
 			[
 				'label' => esc_html__( 'Weight', 'elementor' ),
 				'type' => Controls_Manager::SLIDER,
+				'size_units' => [ 'px', '%', 'em', 'rem' ],
 				'default' => [
+					'unit' => 'px',
 					'size' => 1,
 				],
 				'range' => [
 					'px' => [
 						'min' => 1,
 						'max' => 20,
 					],
+					'em' => [
+						'max' => 5,
+						'step' => 0.1,
```

Comment:
@rodolphebertozzo The default of the `em` and `rem` is:
```
'em' => [
		'min' => 0.1,
		'max' => 10,
		'step' => 0.1,
	],
'rem' => [
		'min' => 0.1,
		'max' => 10,
		'step' => 0.1,
	],
```
Learn more here: https://github.com/elementor/elementor/blob/master/includes/controls/base-units.php#L46

So you don't need to add this again in the control it self, only if you want to change the default values.

Reply:
Done.